script/version-bump.sh

```diff
@@ -25,11 +25,5 @@ then
     exit 4
 fi
 
-# Bump version to next SNAPSHOT
-sbt incrVersion
-
-"$DIR"/update-changes.sh
-
-DEV_VERSION=$("$DIR"/get-version.sh)
-git add --update
-git commit -m "Bump version to ${DEV_VERSION}"
```

Comment:
It made more sense to move the git stuff back into the main script, I think. Having them in here is a bit of spaghetti.